includes/pages/admin_active.php

```diff
@@ -229,14 +229,30 @@ function admin_active()
             }
         }
 
+        $user_worklogs = $usr->worklogs()
+            ->with(['user', 'creator'])
```

Comment:
That one can be ignored too as its unused data and thus two useless queries for every user